### ISSUE TITLE
Rust needs wrappers for Array, Dictionary and String

### ISSUE THREAD

**Issue by fwsGonzo** [libriscv/godot-sandbox#112]
In order to be usable in unboxed mode, which is now the default, the Rust API needs these abstraction in order to be useful. The Rust API is still fully usable if unboxed mode is disabled in project settings, which makes all function arguments Variant.

**Comment by fire** [comment]
Am I able to configure unboxed mode per sandbox?

**Comment by fwsGonzo** [comment]
Yes, it's actually a run-time setting. I can expose it as a property

**Comment by fire** [comment]
Merged?

**Comment by fwsGonzo** [comment]
There is now a basic GodotArray in the Rust API, https://github.com/libriscv/godot-sandbox/commit/32bf88c0378f5b89a7f143628ce53c517a83f910

**Comment by fwsGonzo** [comment]
There is now a basic GodotDictionary in the Rust API, which closes this issue. It still needs to be documented.
